plugins/modules/docker_compose_v2.py

```diff
@@ -54,14 +54,26 @@
       - V(always) ensures that the images are always pulled, even when already present on the Docker daemon.
       - V(missing) only pulls them when they are not present on the Docker daemon.
       - V(never) never pulls images. If they are not present, the module will fail when trying to create the containers that need them.
-      - V(policy) use the C(pull_policy) defined for the service to figure out what to do.
+      - V(policy) use the Compose file's C(pull_policy) defined for the service to figure out what to do.
     type: str
     choices:
       - always
       - missing
       - never
       - policy
     default: policy
+  build:
+    description:
+      - Whether to build images before starting containers. This is used when O(state=present).
+      - V(always) always builds before starting containers. This is equivalent to the C(--build) option of C(docker compose up).
+      - V(never) never builds before starting containers. This is equivalent to the C(--no-build) option of C(docker compose up).
+      - V(policy) uses the policy as defined in the Copose file.
```

Comment:
Thank you. 

Here's a minor typo (Copose). 

The description for the build option  `This is used when O(state=present).` is different from the pull option: `This is used when C(docker compose up) is ran.`

Reply:
I'm currently not sure which of the two descriptions is better. Probably the one for pull, since `state=stopped` can also use `docker compose up --no-start` to create the containers.

Reply:
I'm adjusting it in #766.